qa/tasks/cephfs/test_volumes.py

```diff
@@ -600,6 +708,37 @@ def test_nonexistent_subvolume_group_snapshot_rm(self):
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
 
+    def test_subvolume_group_snapshot_ls(self):
+        # tests the 'fs subvolumegroup snapshot ls' command
+
+        snapshots = []
+
+        # create group
+        group = self._generate_random_group_name()
+        self._fs_cmd("subvolumegroup", "create", self.volname, group)
+
+        # create subvolumegroup snapshots
+        snapshot1 = self._generate_random_snapshot_name()
```

Comment:
No need to repeat the commands three times. Do this in a loop as mentioned above.